pkg/vfs/vfs.go

```diff
@@ -62,13 +62,14 @@ type Config struct {
 	AccessLog            string `json:",omitempty"`
 	PrefixInternal       bool
 	HideInternal         bool
-	RootSquash           *RootSquash `json:",omitempty"`
+	UserSquash           *UserSquash `json:",omitempty"`
```

Comment:
Will it break anything (loading a JSON)?

Reply:
Usually it's dynamically loaded